src/more-infos/more-info-media_player.js

```diff
@@ -70,12 +70,34 @@ export default new Polymer({
       value: false,
     },
 
+    supportsVolumeButtons: {
+      type: Boolean,
+      value: false,
+    },
+
+    hasMediaControl: {
+      type: Boolean,
+      value: false,
+    },
+
+  },
+
+  attached() {
+    // This is required to bind a mousedown event in all browsers
+    const _this = this;
+    this.$.volumeUp.onmousedown = function onVolumeUpDown() {_this.handleVolumeUp();};
```

Comment:
If you want to use JavaScript to attach event listeners, use this approach:

```javascript
this.$.volumeUp.addEventListener('mousedown', () => this.handleVolumeUp());
```

That being said - you can just use the template to attach event listeners, that is the preferred method:

```html
<paper-icon-button id="volumeDown" disabled$='[[isMuted]]'
  icon="mdi:volume-medium" on-mousedown="handleVolumeUp"></paper-icon-button>
```

That being said 2 - we don't have any code within Home Assistant that currently works by keeping it pressed and it is seldom seen in touch UIs. We shouldn't do this.